scripts/update_hardcoded_specs.py

```diff
@@ -70,15 +64,16 @@ def main(migrate_genesis: bool):
         # remove side-products
         os.remove(f'{new_file_base}.json')
         os.remove(f'{new_file_base}-raw.json')
+        os.remove(f'{new_file_base}-raw-unsorted.json')
         os.remove(f'{new_file_base}.state')
         os.remove(f'{new_file_base}.wasm')
 
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--migrate-genesis', help='Create entirely new chain spec, not preserving previous genesis state', action='store_true')
+    parser.add_argument('--regenesis', help='Overwrite genesis state in chain spec. Use this for resetting chains entirely', action='store_true')
```

Comment:
this I just copied over from encointer which fixed the misunderstanding of the flag